lib/cli-engine/cli-engine.js

```diff
@@ -530,7 +529,6 @@ class CLIEngine {
     /**
      * Creates a new instance of the core CLI engine.
      * @param {CLIEngineOptions} providedOptions The options for this instance.
-     * @constructor
```

Comment:
ES2015 classes no longer need to be documented as `@class` or `@constructor` - the parser can now identify this case.